### ISSUE TITLE
Unify and cleanup HTML signature

### ISSUE THREAD

**Issue by stklcode** [pluginkollektiv/cachify#108]
Cachify provides a signature as HTML comment below the cached markup. This signature is uniform for HDD, Memcached and APC, but the DB cache shows a different format with additional information.

<details> 
<summary>Example HDD cache</summary>
<pre>
&lt;!-- Cachify | http://cachify.de
HDD Cache @ 01.01.2017 01:23:45 -->
</pre>
</details>

<details> 
<summary>Example DB cache</summary>
<pre>
&lt;!--
Cachify | http://cachify.de
Without Cachify: 50 DB queries, 0.10 seconds, 23,45 MB
With Cachify: 20 DB queries, 0.05 seconds, 12,34 MB
Generated: 10 hours ago
-->
</pre>
</details>

<br/>

**I got two points here:**

1. I don't see any real advantage of calculating the timespan for the DB backend, despite "because we can", so my suggestion is to unify the strings.
2. I assume that >99% of the page visitors don't care about caching at all and also most administrators don't check the query/memory details regularily. Details are OK for setup and debugging, but most time pointless in production environments. (some might also claim an information leak)
So my second suggestion is to make those details optional and print only minimal information.

With both changes, signatures would look like following

**Signature with details (old behavior, default?)**
```
<!-- Cachify | http://cachify.de
{METHOD} @ {DATE TIME}
{DETAILS}
-->
```

**Cleaned signature:**
```
<!-- Cachify | http://cachify.de
Generated @ {DATE TIME} -->
```

Open for discussion here. Would those changes be welcome?

If so, I got a branch ready for PR ( stklcode/cachify@31ea8ed6e6ec653ce89b6dd03facbd6f54860b30, stklcode/cachify@42c7ab99f36c5901dc376c9bf391633559b4c746 ).

**Comment by krafit** [comment]
Thank you, @stklcode! You are right, I too see no point in providing additional information for DB caches. 
Please go ahead and open a PR.

**Comment by stklcode** [comment]
Just one point I'm curious about. Right now the default behavior is the old (full) signature.

I didn't want to modify this without feedback, but swapping the default is obviously only one character at two positions ;)

**Comment by krafit** [comment]
imho the new (slim) signature should be default

**Comment by stklcode** [comment]
I totally agree. Swapped the default (and restored the changes on the settings page, I accidentially reverted...)

**Comment by swissspidy** [comment]
Why even add anything to the HTML footer at all? Doesn't an HTTP header suffice?

**Comment by stklcode** [comment]
It's there ever since Cachify has been ported to Git... Probably just advertising your plugin and providing easily accessible information about the age of the content.

Moving the information to a header flag might be easy for DB caching, but introduces overhead in the web server configuration. Also many users manage to open and read the page source, but are not familiar with a debugging console to see headers.

Pondering the data overhead between two body lines and a header flag, the HTML comment introduces a slightly smaller footprint in HTTP 1.x environments (no header compression) - but talking about a few byte in the context of a WP page should not be that much of an argument.

I personally would not spend too much additional effort in refactoring this.

**Comment by krafit** [comment]
I'm with @stklcode on this one. The HTML comment is common among caching plugins and users learned to read and understand it (I guess mostly for support) 

Let's stick with HTML for now

**Comment by websupporter** [comment]
merged in #109
